### PR TITLE
Split the badge, notification, and push article in 2

### LINKED ISSUE #2591: Re-title and/or split notifications, badges, and push page?
**Link to the documentation page**: https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges

**Description of the issue**: These three concepts are loosely related, but combining them makes it difficult to find details on push. I always end up looking at the background sync doc trying to find it. Also the title is much more verbose than other similar pages.

AB#44676504

### PR DESCRIPTION
**Pull request by captainbrosset**
Fixes #2591.

The idea of this PR is that notifications, badges, and push messages are 3 separate topics.
While notifications and badges make sense to treat together, because they are 2 ways to alert the user's attention, push messages are a different thing altogether.
Many people will come to the documentation looking for docs on push messages, and might not find it because it's nested within another article.

So, I'm extracting the push messages docs into its own md file.

* Preview links:
  * [Notifications and badges](https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges?branch=pr-en-us-2652)
  * [Push](https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/push?branch=pr-en-us-2652)
* GH rendered markdown:
  * [Notifications and badges](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/split-push/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md)
  * [Push](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/split-push/microsoft-edge/progressive-web-apps-chromium/how-to/push.md)

AB#44676504

### CODE REVIEW

File: microsoft-edge/toc.yml [hunk 1]
```diff
@@ -1067,9 +1067,12 @@
             - name: Define icons and a theme color
               href: progressive-web-apps-chromium/how-to/icon-theme-color.md
 
-            - name: Re-engage users with badges, notifications, and push messages
+            - name: Re-engage users with badges and notifications
               href: progressive-web-apps-chromium/how-to/notifications-badges.md
```
Comment (mikehoffms) [inline microsoft-edge/toc.yml:1071]:
```suggestion
              href: progressive-web-apps-chromium/how-to/notifications-badges.md
              display: Re-engage users with badges, notifications, and push messages  # old title
```
optional, this is a synonym Index entry: if you paste into upper left "Filter by title" box, will find article by using old title.

File: microsoft-edge/toc.yml [hunk 2]
```diff
@@ -1067,9 +1067,12 @@
...
 
+            - name: Re-engage users with push messages
+              href: progressive-web-apps-chromium/how-to/push.md
```
Comment (mikehoffms) [inline microsoft-edge/toc.yml:1074]:
```suggestion
              href: progressive-web-apps-chromium/how-to/push.md
              displayName: Re-engage users with badges, notifications, and push messages  # old top-of-page title
```
optional, this is a synonym Index entry: if you paste into upper left "Filter by title" box, will find article by using old title. see other instances of "displayName:" in toc file.

**Comment by mikehoffms** [comment]
GH rendered markdown (non-commit-specific links):
Notifications and badges:
https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/split-push/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md
Push:
https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/split-push/microsoft-edge/progressive-web-apps-chromium/how-to/push.md

**Review by mikehoffms — APPROVED** [review]

**Review by Reezaali — APPROVED** [review]
Approving to unblock the merge.

**Pull request merged**